src/App.tsx

```diff
@@ -78,6 +78,25 @@ class Board extends React.Component<BoardProps, BoardState> {
         }
     }
 
+    deleteSelectedSquares() : void {
+        for (let i = 0; i < this.state.highlights.length; i++) {
+            let addy : SquareAddress = this.state.highlights[i];
+            if (!this.state.permanentValues.includes(addy)) {
+                let newValues = JSON.parse(JSON.stringify(this.state.values));
+                if (newValues[addy]) {
```

Comment:
This if is unnecessary

Reply:
its to make sure that you don't delete permanent values. You don't want to be able to delete the values passed into the board.

Reply:
oh i see